### PR TITLE
Allow to use LeptonX lite as the UI theme

### LINKED ISSUE #12650: Allow to use LeptonX lite as the UI theme
With the ABP 5.4, we are planning to allow us to select the UI theme while creating a new application.

- [x] Should work for Application and Application (Single-Layer) startup templates with all UI options (MVC, Angular, Blazor WASM and Blazor Server)
- [x] Add `--theme` parameter to ABP CLI. Default value will be `leptonx-lite`. Possible values: `leptonx-lite` and `basic`.
- [x] Add a "Select UI Theme" section to the https://abp.io/get-started page to allow to choose one of the theme options (LeptonX-Lite (selected by default and in the first order) and Basic). Show screenshots of the themes side by side and allow to select by clicking the image. Write explanations under the themes, for leptonx-lite: "A modern and stylish Bootstrap UI theme. Ideal if you want to have a production ready UI theme.", for basic theme "Minimalist UI theme with plain Bootstrap colors and styles. Ideal if you will build your own UI theme."

### PR DESCRIPTION
**Pull request by EngincanV**
Resolves #12650

> After https://github.com/volosoft/lepton/issues/446 is fixed, we need to update the package.json files for `app` and `app-nolayers` templates. (I'll do that)
 
---

### With `--theme basic` option

basic

### Default theme (and `--theme leptonx-lite` option)

leptonx-lite

---

For testing locally;

1. Build the `Volo.Abp.Cli` project.
2. Zip the `app` and `app-nolayers` templates in this branch and
3. Replace with the latest template zip files in the cache. (default path is for windows: `%USERPROFILE%\.abp\templates` )
4. Create a new project with the command below:
   
```bash
D:\GitHub\abp\framework\src\Volo.Abp.Cli\bin\Debug\net6.0\Volo.Abp.Cli.dll new LptnXLtDemo -t app -u blazor --theme basic
```

> The above command will create an app template with Blazor UI and Basic Theme. You can also test for the other UI types and themes.

---

### CODE REVIEW

**Comment by berkansasmaz** [comment]
It may be necessary to add theme information to usage info.

https://github.com/abpframework/abp/blob/6d5b438d303a8fb7241f6ff59fb8f1bfda0f85ba/framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/Commands/NewCommand.cs#L90-L140

**Comment by maliming** [comment]
We use two themes in the templates, since lepton is the default, we can remove the basic theme.

Rename lepton(`.cs, .csproj, .json`) to basic if the developer prefer to use basic. I believe that few people will choose basic in the future.

**Comment by hikalkan** [comment]
I also assigned @gizemmutukurt to this, so she manually tests after our review.

**Review by hikalkan — APPROVED** [review]

**Comment by hikalkan** [comment]
@maliming I reviewed, but not so deep. You review and approve, then @gizemmutukurt will make manual tests too.

File: framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Templates/App/AppTemplateBase.cs [hunk 1]
```diff
@@ -33,6 +33,7 @@ public override IEnumerable<ProjectBuildPipelineStep> GetCustomSteps(ProjectBuil
         RemoveMigrations(context, steps);
         ConfigureTieredArchitecture(context, steps);
         ConfigurePublicWebSite(context, steps);
+        ConfigureTheme(context, steps);
```
Comment (maliming) [inline framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Templates/App/AppTemplateBase.cs:36]: I think only free app template support to replace the theme, not the commercial template.

**Review by maliming — COMMENTED** [review]
We will do the test again when the npm packages are released.

**Review by maliming — APPROVED** [review]

**Pull request merged**